### PR TITLE
Remove dead mirrors from mirrorchecker config

### PR DESCRIPTION
**Pull request by merlyel**

### CODE REVIEW

**Comment by merlyel** [comment]
waiting for update from massclouds admin

File: configs/mirrors.txt [hunk 1]
```diff
@@ -1,14 +1,11 @@
 url='http://resources.ovirt.org/pub',whitelist=True
-url='http://mirror.linux.duke.edu/ovirt/pub'
 url='http://mirrors.ibiblio.org/ovirt'
 url='http://mirror.isoc.org.il/pub/ovirt'
-url='http://www.gtlib.gatech.edu/pub/oVirt/pub'
```
Comment (sandrobonazzola) [inline configs/mirrors.txt:5]: this one is still reachable but outdated, perhaps the syncing is not working anymore?

File: configs/mirrors.txt [hunk 2]
```diff
@@ -1,14 +1,11 @@
 url='http://resources.ovirt.org/pub',whitelist=True
-url='http://mirror.linux.duke.edu/ovirt/pub'
```
Comment (sandrobonazzola) [inline configs/mirrors.txt:2]: this one is still reachable but outdated, perhaps the syncing is not working anymore?

File: configs/mirrors.txt [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
 url='http://mirror.isoc.org.il/pub/ovirt'
-url='http://www.gtlib.gatech.edu/pub/oVirt/pub'
 url='http://ftp.plusline.net/ovirt'
 url='http://mirrors.rit.edu/ovirt'
 url='http://mirror.slu.cz/ovirt'
 url='http://ftp.snt.utwente.nl/pub/software/ovirt'
 url='http://ftp.nluug.nl/os/Linux/virtual/ovirt'
 url='http://ovirt.repo.nfrance.com'
 url='http://mirror.rackspace.com/ovirt'
-url='http://mirror.massclouds.com/ovirt'
```
Comment (sandrobonazzola) [inline configs/mirrors.txt]: this one doesn't work for me as well

**Comment by sandrobonazzola** [comment]
Missing `Signed-Off-By` in the commit message

File: configs/mirrors.txt (hunk 1 above)
Comment (merlyel) [reply configs/mirrors.txt:5]: Maybe there problem is just with sync, but admins haven't been answering for a long time already, so I don't see any other options than to drop them from the mirrorslist

File: configs/mirrors.txt (hunk 3 above)
Comment (merlyel) [reply configs/mirrors.txt]: I wrote to admin yesterday, so I wanted to wait for their answer for a day or two. That's why MR is draft state

File: configs/mirrors.txt (hunk 2 above)
Comment (merlyel) [reply configs/mirrors.txt:2]: Same as for gatech - no replies from them

Comment (sandrobonazzola) [reply configs/mirrors.txt:2]: ok, +1 for removal then

File: configs/mirrors.txt (hunk 3 above)
Comment (sandrobonazzola) [reply configs/mirrors.txt]: ok makes sense

File: configs/mirrors.txt (hunk 1 above)
Comment (sandrobonazzola) [reply configs/mirrors.txt:5]: ok

**Review by sandrobonazzola — APPROVED** [review]

**Review by sandrobonazzola — APPROVED** [review]

File: configs/mirrors.txt (hunk 3 above)
Comment (merlyel) [reply configs/mirrors.txt]: Massclouds admin replied finally - they had a problem due to migration of the server, sync is restored. I restored this mirror in configs

**Pull request merged**